x-pack/metricbeat/module/oracle/sysmetric/sysmetric_metric.go

```diff
@@ -0,0 +1,89 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package sysmetric
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strconv"
+
+	"github.com/elastic/beats/v7/x-pack/metricbeat/module/oracle"
+	"github.com/elastic/elastic-agent-libs/mapstr"
+)
+
+type sysmetricMetric struct {
+	beginTime   sql.NullString
+	endTime     sql.NullString
+	intsizeCsec sql.NullFloat64
+	groupId     sql.NullInt64
+	metricId    sql.NullInt64
+	name        sql.NullString
+	value       sql.NullFloat64
+	metricUnit  sql.NullString
+	conId       sql.NullFloat64
+}
+
+/*
+ * The following function executes a query that produces the following result
+ *
+ * BEGIN_TIM END_TIME  INTSIZE_CSEC GROUP_ID  METRIC_ID METRIC_NAME				VALUE	   METRIC_UNIT		   CON_ID
+ * 19-APR-22 19-APR-22         6042        2       2146 I/O Requests per Second 2.99569679 Requests per Second 0                                                      0
+ *
+ * Which is parsed into sysmetricMetric instances
+ */
+func (e *sysmetricExtractor) calQuery() string {
```

Comment:
```suggestion
func (e *sysmetricExtractor) callQuery() string {
```

I suspect there is a missing `l` here

Reply:
`calQuery` was actually supposed to mean "calculate query". Should we rename it to `calculateQuery` or do you suggest rephrasing it?

Reply:
Ah yes `calculateQuery` is fine. Thanks. `calc` is a more frequent abbreviation for `calculate` than `cal`.

Reply:
That makes sense, we'll make sure to use the same going ahead. Thanks!